sdk/search/Azure.Search.Documents/tests/DocumentOperations/IndexingTests.cs

```diff
@@ -848,7 +847,18 @@ public async Task ThrowsOnInvalidDocument()
                     batch,
                     new IndexDocumentsOptions { ThrowOnAnyError = true }));
             Assert.AreEqual(400, ex.Status);
-            StringAssert.StartsWith("The request is invalid. Details: actions : 0: Document key cannot be missing or empty.", ex.Message);
```

Comment:
This is why we don't expose details of our error model...  I'll do dumb things that shouldn't be relied on.

Is the new error returned for all versions of the service or just the latest?  We might need to put a [ServiceVersion] attribute on the test if it's only the latest.

Reply:
The new error format is the one supported since "2020-06-30".